Automattic/wp-calypso
# Monthly default experiment: Remove experiment code

#### Changes proposed in this Pull Request

* Removes experiment code introduced in https://github.com/Automattic/wp-calypso/pull/60182 and https://github.com/Automattic/wp-calypso/pull/59771. Control experience is retained.

#### Testing instructions



* Go through the signup flow and verify that you are able to complete purchase of a plan. No JS errors should be seen in the console.
* On an account which has the legacy plans, verify that the logged-in `/plans` page in Calypso loads without JS errors, and that you are able to upgrade to a higher (legacy) plan.
